apps/webapp/src/components/swap/swap-form.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from '@penumbra-zone/ui';
+import { useToast } from '@penumbra-zone/ui/components/ui/use-toast';
+import InputToken from '../shared/input-token.tsx';
+import { useLoaderData } from 'react-router-dom';
+import { AccountBalance } from '../../fetchers/balances.ts';
+import { useStore } from '../../state';
+import { swapSelector } from '../../state/swap.ts';
+import { AssetOutBox } from './asset-out-box.tsx';
+
+export const SwapForm = () => {
```

Comment:
I can't actually navigate to this UI in my local git repo to test it, since I'm getting the "missing trailer" issue we're discussing in Discord. But the code looks fine!